lib/helpers.js

```diff
@@ -171,6 +171,33 @@ function validatePrecacheInput(items) {
   return items;
 }
 
+function isResponseFresh(response, maxAgeSeconds, now) {
+  // If we don't have a response, then it's not fresh.
+  if (!response) {
+    return false;
+  }
+
+  // Only bother checking the age of the response if maxAgeSeconds is set.
+  if (maxAgeSeconds) {
+    var dateHeader = response.headers.get('date');
+    // If there's no Date: header, then fall through and return true.
+    if (dateHeader) {
+      var parsedDate = new Date(dateHeader);
+      // If the Date: header was invalid for some reason, parsedDate.getTime()
+      // will return NaN, and the comparison will always be false. That means
+      // that an invalid date will be treated as if the response is fresh.
+      if ((parsedDate.getTime() + maxAgeSeconds) < now) {
```

Comment:
Since its "maxAgeSeconds",  it should be `(parsedDate.getTime() + maxAgeSeconds * 1000) < now)`